docs/source/ContributionGuide.rst

```diff
@@ -242,6 +242,38 @@ approved by your employer) and you grant a full, complete, irrevocable
 copyright license to all users and developers of the Avocado project,
 present and future, pursuant to the license of the project.
 
+Code Review
+-----------
+
+Every single Pull Request in Avocado has to be reviewed by at least one
+other developer. All members of the core team have permission to merge
+a Pull Request, but there are some conditions that have to be fulfilled
+before merging the code:
+
+- Pull Request has to pass the CI tests.
+- One 'Approved' code review should be given.
+- No explicit disapproval should be present.
```

Comment:
I agree with "No explicit disapproval".  A NACK should be given when something is clearly broken and will disrupt the project.  It should never be given for situations where one sees an alternative implementation, if it has some benefits.  For instance, optmizing code for better (less) CPU usage in non-critical areas should *not* be a reson for a NACK.